### DISCUSSION TITLE
Just wanted to say--nice job you guys!

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#742] [General]
I'm up and running with REM now and just wanted to say how impressed I am with all you've done!

I've done several techy type projects around the house, but this one has shown me a few new things hardware wise:

1)  DIN rail and terminal blocks

I knew what a DIN rail was, but thought it was pretty much a European thing and never tried it myself.  Previously, I've laser cut a piece of acrylic and used holes, machine screws, and nuts to secure  stuff on the acrylic.  This is way better!

It's embarrassing, butI had to watch a video on DIN terminal blocks to learn that the shorting plugs make a bus.   So, I only need three terminal blocks to distribute a one in, five out bus, not six!  (Duh) 

2)  Ferrules and tool to crimp them.

Awesome!  Didn't know how easy it was to get a good crimp, or maybe it's the tool you recommended.  Way nicer than poking stranded wire into hole, especially when you don't have a good view of the hole.

Thanks for all your hard work!

I plan to do a pcb that should be a big cost reduction compared to Whitepage + EZ PH and EZ ORP + relay board.  I'll post that for comments before I send Gerber for fabrication.

**Comment by tagyoureit** [comment]
Thanks for the compliments!  Much appreciated.
